prow/jobs/jinja/presubmits/e2e_test.jinja2

```diff
@@ -0,0 +1,32 @@
+{% for service in aws_services  %}
+  aws-controllers-k8s/{{ service }}-controller:
+  - name: {{ service }}-pre-submit
+    decorate: true
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    extra_refs:
+    - org: aws-controllers-k8s
+      repo: community
+      base_ref: main
+      workdir: true
+    - org: aws-controllers-k8s
+      repo: test-infra
+      base_ref: main
+      workdir: false
+    - org: aws-controllers-k8s
```

Comment:
is the controller under test needed as extra ref? I thought it would always be checked out.

Reply:
Ah good catch, you're 100% correct.